### ISSUE TITLE
Indexing indicator in status bar.

### ISSUE THREAD

**Issue by yamenarahman** [felixfbecker/vscode-php-intellisense#294]
I think we should implement an indexing indicator so it tells the progress of indexing. I can help with PR but don't know from where to start.
Thanks for everything <3

**Comment by RobberPhex** [comment]
I am working on PHP project with 17499 `*.php` files.

I really want this plugin can indicate that `how many files have been indexed` and `how many files remained to be processed`.

**Comment by felixfbecker** [comment]
VS Code doesn’t have this capability yet. See https://github.com/Microsoft/vscode-languageserver-node/pull/261.

**Comment by RobberPhex** [comment]
@felixfbecker 
Yep, LSP doesn't support this. But the extension can indicate progress.
For example, [vscode-intelephense](https://github.com/bmewburn/vscode-intelephense/blob/e5a02b73653d64379d19f932cf1ecc12e817ee25/src/extension.ts#L101) shows "indexing" while indexing.

**Comment by felixfbecker** [comment]
Yes, and it does that through non-standard LSP methods, which I don’t want to add to the language server because no other client than the VS Code extension will understand it. Please express your need for this on the linked issue.

**Comment by ElectricRCAircraftGuy** [comment]
Please add this. I really want to see the indexing progress.
